docs/src/modules/javascript/partials/json.adoc

```diff
@@ -1,24 +1,3 @@
-include::ROOT:partial$include.adoc[]
-
-Kalix integrates with https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] to enable asynchronous messaging. Consumers of published messages are guaranteed to receive them at least once. This means that receivers must be able to handle duplicate messages. If you are new to the world of Pub/Sub, many online resources provide tips on handling duplicate messages.
-
-This page describes how to subscribe to published events, but you also need to configure Google Cloud Pub/Sub access for your Kalix project as explained in https://docs.kalix.io/projects/message-brokers.html[How to configure a broker].
-
-NOTE: It is your responsibility to create the topics in Google Cloud Pub/Sub before configuring publishers or subscribers.
-
-== Subscribing to a topic's messages
-
-To receive messages from a Pub/Sub topic, annotate a service method in the Protobuf service definition with the `(kalix.method).eventing` annotation. Specify the topic name in the `in` section of the annotation:
-
-[source,protobuf,indent=0]
-----
-include::example$eventing-shopping-cart/proto/cart/shopping_cart_analytics.proto[tag=subscribing]
-----
-<1> annotate the Protobuf rpc method with `(kalix.method).eventing`
-<2> use `in` and `topic` to subscribe to a topic
-
-There is nothing specific required in the implementation of `ProcessAdded`. The implementation will in most cases be an Action and forward a converted message to a different component (eg. an Event Sourced Entity).
-
 == Receiving JSON messages
```

Comment:
This section seems to have been dropped in https://github.com/lightbend/kalix-jvm-sdk/pull/373, but I figured it might be useful to have (but might make more sense in the 'serialization' section)?